butane/Cargo.toml

```diff
@@ -12,7 +12,8 @@ repository = "https://github.com/Electron100/butane"
 documentation = "https://docs.rs/butane/"
 
 [features]
-default = ["datetime", "uuid"]
+default = ["datetime", "json", "uuid"]
```

Comment:
I've enabled it by default as serde_json is already a dependency of butane, and this gives butane the ability store almost any data structure, even if it isnt in the format that they want it stored it.

Reply:
Fair enough